Makefile

```diff
@@ -225,10 +236,37 @@ docker-push-init-users: docker-build-init-users
 docker-push-webhook: docker-build-webhook
 	docker push ${WEBHOOK_IMG}:${GIT_VERSION}
 
+# Buildx the docker image
+docker-buildx-dataset-controller: generate gen-openapi fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.dataset -t ${DATASET_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-application-controller: generate fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.application -t ${APPLICATION_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-alluxioruntime-controller: generate gen-openapi fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.alluxioruntime -t ${ALLUXIORUNTIME_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-jindoruntime-controller: generate gen-openapi fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.jindoruntime -t ${JINDORUNTIME_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-goosefsruntime-controller: generate gen-openapi fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.goosefsruntime -t ${GOOSEFSRUNTIME_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-juicefsruntime-controller: generate gen-openapi fmt vet juicefsruntime-controller-build
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.juicefsruntime -t ${JUICEFSRUNTIME_CONTROLLER_IMG}:${GIT_VERSION}
+
+docker-buildx-csi: generate fmt vet
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.csi -t ${CSI_IMG}:${GIT_VERSION}
+
+docker-buildx-init-users:
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache charts/alluxio/docker/init-users -t ${INIT_USERS_IMG}:${GIT_VERSION}
 
+docker-buildx-webhook:
+	docker buildx build --push --platform linux/amd64,linux/arm64 --no-cache . -f docker/Dockerfile.webhook -t ${WEBHOOK_IMG}:${GIT_VERSION}
 
 docker-build-all: ${DOCKER_BUILD}
 docker-push-all: ${DOCKER_PUSH}
+docker-buildx-all: ${DOCKER_BUILDX}
```

Comment:
When building multi-platform images using buildx, we need to create a builder for it, refer to https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images

I think `docker-buildx-all` better to add the whole process.

Furthermore, maybe name `docker-buildx-all-push` is more suitable for it.

Reply:
Docker buildx uses qemu emulation to build images for other platforms than the host. 

I already have docker installed on my system. When I built multi-arch images, I simply had to run qemu emulation before using docker buildx, as below: 
~~~ 
$ docker run --rm --privileged multiarch/qemu-user-static --reset -p yes 
$ docker buildx build --platform linux/amd64,linux/arm64 ….. 
~~~ 

You want me to add qemu installation along with the docker buildx commands...am I correct?

Reply:
I think we can manually  run qemu emulation in short time. And Let the users know through documentation.
In long time I agree that we can run `make docker-buildx-all-push` from scratch. I see the similar implementation in https://github.com/openyurtio/openyurt/blob/master/Makefile#L129-L136 .

WDYT? @odidev  and @zwwhdls

Reply:
Yes, I agree. We can add a point in how_to_develop.md to run qemu emulation before using docker buildx build.

Reply:
Would you mind to rename `docker-buildx-all` to `docker-buildx-all-push` ? I think it's more clear for the developer. Thanks.

Reply:
agree

Reply:
@cheyang  Sure, added.

Reply:
Thanks very much!